MAINTAINERS.md

```diff
@@ -4,45 +4,58 @@ Minio Java SDK uses [gradle](https://gradle.org/) build system.
 ## Responsibilities
 Go through [Maintainer Responsibility Guide](https://gist.github.com/abperiasamy/f4d9b31d3186bbd26522).
 
-### Setup your minio-java Github Repository
+## Setup your minio-java Github Repository
 Fork [minio-java](https://github.com/minio/minio-java/fork) source repository to your own personal repository.
-```bash
+```sh
```

Comment:
`bash` and `sh` do not make any difference in markdown

Reply:
sh is preferred over `bash` will stick to `sh`

Reply:
```sh is fine.